data/array/item deleter/internal/37f820aa-4842-43ab-acd4-2c47f999ce08.json

```diff
@@ -9,7 +9,7 @@
     },
     "outputs": {
       "63ede1b7-2b35-42ec-9116-725f7909d3f6": {
-        "type": "typeof(array & index)[\"array\"]",
+        "type": "(typeof `array & index`)[\"array\"]",
```

Comment:
I think it should simply be ```typeof `array` ``` as deletion does not change the type

Reply:
It's an internal, so there is no standalone `array`. It must be extracted from `array & index`.